interceptors/src/main/java/uk/gov/moj/sdt/interceptors/enricher/SubmitQueryEnricher.java

```diff
@@ -105,18 +110,36 @@ public String enrichXml(final String message) {
             }
 
             if (LOGGER.isDebugEnabled()) {
-                LOGGER.debug("Message after enrichment [" + newXml + "]");
+                LOGGER.debug("Message after enrichment [{}]", newXml);
             }
 
         } else {
             // Failure to find matching request in outgoing XML.
             if (LOGGER.isDebugEnabled()) {
-                LOGGER.debug("Parent tag [" + this.getParentTag() + "] not found...skipping enrichment.");
+                LOGGER.debug("Parent tag [{}] not found...skipping enrichment.", this.getParentTag());
             }
         }
 
         return newXml;
 
     }
 
+    private String loadSummaryResults(String newXml) {
+
+        // Get the system specific response from thread local to inject into the outbound message
+        String outXml = newXml;
+
+        // TODO: regular expression replace the existing results node
+        String resultsXml = "<qresp:results/>";
+
+        String resultsXmlNew = SdtContext.getContext().getClaimDefencesSummaryResultsXml();
+
+        if (null != outXml && !outXml.isEmpty() && outXml.contains(resultsXml)) {
+            outXml = outXml.replace(resultsXml, resultsXmlNew);
+            SdtContext.getContext().setRawOutXml(outXml);
```

Comment:
better we remove this from here

Reply:
Thinking about where to put it. Was going to put it into the ResponsesSummaryUtil class but then it would need to be static method which wouldn't be great.